README.md

```diff
@@ -14,16 +14,45 @@ Here is a small share of the more major features that this mod adds to the origi
 * Personal crafting menu
 * Beds
 * Mob Spawners (with loot chests)
-* Expanded storyline: 4th dungeon level, and Second Boss
+* An expanded storyline: 4th dungeon level, and Second Boss
 * Bigger/Resizable screen size
 * Key Binding customization, so you can change the key controls
 * Tutorials
 * Texture Packs
 * And best of all, active development and community!
+* Localization: Different languages
+
+You can find a more comprehensive list of the earlier versions, before 1.9, in [this minecraft forum post](http://www.minecraftforum.net/forums/off-topic/general-gaming/452036-v1-6-minicraft-plus). Be aware however, that it's old; it's from many years ago before the project was abandoned.
```

Comment:
you've added this, but haven't removed the other mention of it... why is this here?